builder/build-debs.sh

```diff
@@ -4,6 +4,8 @@
 
 set -euxo pipefail
 
+git log -1 --oneline
```

Comment:
```suggestion
git log -1 --oneline --no-color
```

Reply:
Thanks!  Fixed in 182468bf6d57f85cffbf19c5655d843a82be8614.  I've also made `--show-signature` explicit, since I realized that's implicit in my `.gitconfig`.